lib/Segments/WooCommerce.php

```diff
@@ -247,7 +259,7 @@ private function updateFirstNames($subscriberId = null) {
     global $wpdb;
     $collate = '';
     if ($this->needsCollationChange()) {
-      $collate = ' COLLATE ' . $this->mailpoetEmailCollation;
+        $collate = ' COLLATE ' . $this->mailpoetEmailCollation;
```

Comment:
I think this change is not intended or is it?

Reply:
I've fixed it with a fixup commit.